File: C++/minimum-number-of-moves-to-seat-everyone.cpp
class Solution {
public:
    int minMovesToSeat(vector<int>& seats, vector<int>& students) {
        sort(seats.begin(), seats.end());
        sort(students.begin(), students.end());
        int sz = seats.size();
        int moves = 0;
        for (int i = 0; i < sz; i++) {
            moves += abs(seats[i] - students[i]);
        }
        return moves;
    }
};

File: Go/minimum-number-of-moves-to-seat-everyone.go
func minMovesToSeat(seats []int, students []int) int {
	slices.Sort(seats)
	slices.Sort(students)
	moves := 0
	for i := range seats {
		moves += abs(seats[i] - students[i])
	}
	return moves
}

func abs(x int) int {
	if x < 0 {
		return -x
	}
	return x
}